### DISCUSSION TITLE
Tagging the maintainer - tl;dr: don't

### DISCUSSION THREAD

**Discussion by Lockszmith-GH** [zellij-org/zellij#3794] [General]
### Welcome
If you're new to the community, welcome!

_If you've been referred to this article, I hope you will take it at face value and not take offense._

From my personal experience, the community is a very welcoming community, I've been lurking in the shadows for a long time before I started asking questions, I dare request anything without thoroughly thinking it through and considering the impact, because I see how busy the different maintainers are.

Nobody asked me to write this, but I feel there should be a shortcut to this type of blurb, so I'm taking a presumptuous step and writing this in response to a comment tagging **imsnif**. Hopefully this will be the last time this is referred to, but just in case it will be needed again, this will be available as #3794

### Please read this

At the time of writing this, the project has 959 open issues and 77 open PRs and 21.9K stars - it's a highly busy, popular project by any standard, and **imsnif** the maintainer (note that I did not tag him, just emphasized his nickname) is doing this full-time (I recommend reading [his blog post](https://poor.dev/blog/why-zellij/)).

To quote <b>@</b>**imsnif** himself from a week ago in the closing comment of [PR #3437](https://github.com/zellij-org/zellij/pull/3437#issuecomment-2467571534):

> ...maintainers of popular projects get an extremely large volume of notifications we need to parse through from various avenues. Please do not ever tag a maintainer personally for anything unless it is absolutely urgent. Thanks!

If you want to actually contribute, start with [CONTRIBUTING.md](https://github.com/zellij-org/zellij/blob/main/CONTRIBUTING.md#status-of-code-contributions-prs) and the project's [roadmap](https://zellij.dev/roadmap/). Anything out of that scope takes time away from planned action.

If you have a question, ask the community, this is what the discussions are for.

But please, unless this is critical in terms of 'everything, everywhere, is broken' or there is a critical security issue, don't tag a maintainer just because you have an itch to scratch. They are doing a thankless job already, let's not add to the burden. They don't owe us anything (This is true even if you decided to support the project financially - and [please do](https://poor.dev)).

### Please acknowledge reading
After reading, please acknowledge with the 👀 emoji to indicate you've read this (And unsubscribe, if you don't want to get notifications)

**Comment by Lockszmith-GH** [comment]
Recommended related reading:
* https://github.com/twpayne/chezmoi/discussions/2819#discussioncomment-5898088

**Comment by giyany** [comment]
Thank you for writing this and for putting in the effort and thought into the zellij community.
I really do hope that people understand that this is a practical matter only. Reports, questions and discussions are all very welcome and helpful. Providing channels for communication and keeping them accessible and safe is something we put a lot of thought into. The reality is that without these rules, a lead maintainer can receive dozes of notifications daily on top of all the other flow on Github, reddit, discord, etc. etc. In a large OSS project it's unfeasible to read everything, let alone work or keep one's peace of mind with many hourly notifications.
As a rather early member of the zellij org, it's been incredible to see the community grow into what it is today. We're very proud of the overall positive atmosphere that zellij users cultivate and we hope it stays this way <3
